src/Resources/views/Admin/Pixel/_form.html.twig

```diff
@@ -0,0 +1,11 @@
+<div class="ui segment">
+    {{ form_errors(form) }}
+    <div class="two fields">
+        {{ form_row(form.pixelId) }}
+        {{ form_row(form.channels) }}
+    </div>
+    {{ form_row(form.enabled) }}
+</div>
+<div class="ui segment">
+    {{ form_row(form.customAccessToken) }}
```

Comment:
I think you should add a help text here

Reply:
To explain that users don't have to input an access token, only if it's different than the default one